intuita-webview/src/shared/Tree/index.tsx

```diff
@@ -13,38 +12,21 @@ type Props = {
 		depth,
 		open,
 		setIsOpen,
-		color,
-		lastChild,
 	}: {
 		index: number;
 		node: TreeNode;
 		depth: number;
 		open: boolean;
 		setIsOpen: (value: boolean) => void;
-		color: string;
-		lastChild: boolean;
 	}): ReactNode;
-	color: string;
-	lastChild: boolean;
 };
 
-const Tree = ({
-	node,
-	depth,
-	renderItem,
-	color: colorProp,
-	index,
-	lastChild,
-}: Props) => {
+const Tree = ({ node, depth, renderItem, index }: Props) => {
 	const hasNoChildren = !node.children || node.children.length === 0;
-	const [open, setIsOpen] = useState(depth === 0);
-	const [color] = useState(
-		hasNoChildren ? colorProp : generateColor(buildHash(node.id)),
-	);
+	const [open, setIsOpen] = useState(true);
```

Comment:
`true` by default because it's easier for user to explore when all folders are expanded